### DISCUSSION TITLE
Chat

### DISCUSSION THREAD

**Discussion by Bizarrelego** [Bizarrelego/Roblox-Mass-Unfavorite-Games#1] [General]
Chat here or ask questions

**Comment by Galambkommando** [comment]
Hi! It does not work anymore, 
![image](https://github.com/user-attachments/assets/800f5c33-c163-47cb-acd5-049aeab0b3c0)

**Reply by Bizarrelego** [reply]
Thanks for the heads up :)

**Comment by flaminisdumb** [comment]
is this safe to use? i wanna clear my favourites but i don't want to risk anything

**Reply by Bizarrelego** [reply]
It's open source, so all code is viewable. Currently though I have not spent any time fixing it as roblox changed the endpoints. I'll work on this sometime soon
